packages/teams-js/src/private/teams.ts

```diff
@@ -82,4 +85,77 @@ export namespace teams {
   export function isSupported(): boolean {
     return runtime.supports.teams ? true : false;
   }
+
+  export namespace fullTrust {
+    export namespace joinedTeams {
+      /**
+       * @hidden
+       * Hide from docs
+       * ------
+       * Allows an app to retrieve information of all user joined teams
+       *
+       * @param teamInstanceParameters - OPTIONAL Flags that specify whether to scope call to favorite teams
```

Comment:
Since you're editing this file, can you please change this toL
`@param teamInstanceParameters - Optional flags that specify...`